src/audiomixerboard.cpp

```diff
@@ -747,8 +747,13 @@ void CChannelFader::SetChannelInfos ( const CChannelInfo& cChanInfo )
     // Country flag icon -------------------------------------------------------
     if ( cChanInfo.eCountry != QLocale::AnyCountry )
     {
+#if QT_VERSION >= QT_VERSION_CHECK( 6, 0, 0 )
+        QLocale::Country eC = CLocale::LocaleQt5toQt6 ( cChanInfo.eCountry );
```

Comment:
Generally, single letter variable names are avoided.  This should be `eCountry`.

Reply:
Thanks, updated.